### ISSUE TITLE
Bot errors out when theres'an uncaptured pokemon (only seen) in your pokedex.

### ISSUE THREAD

**Issue by RedNax67** [NECROBOTIO/NecroBot#1701]
The issue is cause by GetPokeDexCount.

`Pokedex: 59 { "inventoryItemData": { "pokedexEntry": { "pokemonId": "PERSIAN", "timesEncountered": 1 } } }
`

The below fixes it..

 ```
               if (CaughtPokemon.Count() > 1 )
                {
                    var split = CaughtPokemon[1].Split(' ');
                    int Times = int.Parse(split[1]);
                    if (Times > 0)
                        timesCaptured++;
                }

```

**Comment by sawirth** [comment]
this should also fix it and is a much cleaner solution #1646

**Comment by actislav** [comment]
#1706 fixed it in
